dashboard/app/models/script.rb

```diff
@@ -38,15 +38,8 @@ class Script < ActiveRecord::Base
   belongs_to :user
 
   attr_accessor :skip_name_format_validation
+  include SerializedToFileValidation
 
-  validates :name,
-    presence: true,
-    uniqueness: {case_sensitive: false},
-    format: {
-      unless: :skip_name_format_validation,
-      with: /\A[a-z0-9\-]+\z/,
-      message: 'can only contain lowercase letters, numbers and dashes'
-    }
```

Comment:
I had tried removing the following validation as well (cannot start with tilde/dot), since it only comes into play in the case where skip_name_format_validation is false. However, there was a test for that specific situation, so I'm assuming we want to retain that logic.